articles/relnotes/index.md

```diff
@@ -17,19 +17,33 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: February 27th, 2020*
+
+This release contains the following:
+
+- New Quantum Machine Learning library, for more information go to our [QML docs page](https://docs.microsoft.com/quantum/libraries/machine-learning/?view=qsharp-preview)
+- Bug fixes to the Quantum Katas, resulting in a 10-20x performance increase
```

Comment:
It is my understanding that the fixes are not in the katas but in the QDK itself (was it iqsharp?) - the perf improvement just manifests on the katas?

```suggestion
-  IQ# bug fixes, resulting in a 10-20x performance increase on the Quantum Katas
```

Reply:
thanks Mariia!